tests/unit/backend/layers/common/base_test.py

```diff
@@ -80,11 +90,8 @@ def setUp(self):
 
         # Mock CorporaConfig
         # TODO: deduplicate with base_api
-        def mock_config_fn(name):
-            if name == "upload_max_file_size_gb":
-                return 30
```

Comment:
Replace with a dict (mutable, can add `is_cxg_admin` value without overriding the other config values and the result returned from the mock `side_effect`)